muspinsim/tools/generator.py

```diff
@@ -51,21 +58,22 @@ class QuadrupoleIntGeneratorGIPAWOut(InteractionGenerator):
     _gipaw_file: GIPAWOutput
 
     def __init__(self, gipaw_file: GIPAWOutput):
+        super().__init__(num_inputs=1)
+
         self._gipaw_file = gipaw_file
 
-    def gen_config(
-        self, muon_file_index: int, atom_file_index: int, atom: CellAtom
-    ) -> str:
+    def gen_config(self, atom_file_indices: List[int], atoms: List[CellAtom]) -> str:
         # Obtain corresponding EFG tensor
-        gipaw_atom = self._gipaw_file.find_atom(atom.index)
+        gipaw_atom = self._gipaw_file.find_atom(atoms[0].index)
 
         if gipaw_atom is None:
             raise ValueError(
-                f"Unable to locate atom with index {atom.index} in GIPAW output file"
+                f"""Unable to locate atom with index {atoms[0].index} in GIPAW
+output file"""
```

Comment:
Yes, the line gets too long otherwise, it was only atom.index before and the extra few characters makes flake complain.

Reply:
Fair enough, in other places I thought we were using concatenation for the long strings, rather than using `"""..."""` so was just wondering why it was different here. Maybe because it's an `f` string it gets treated by black/flake/whatever differently? Regardless it's of little consequence, was curious more t han anything else.